cosmo_tester/test_suites/image_based_tests/ssl_test.py

```diff
@@ -14,18 +14,22 @@
 #    * limitations under the License.
 
 from cosmo_tester.framework.examples.hello_world import centos_hello_world
-from cosmo_tester.framework.fixtures import bootstrap_based_manager
+from cosmo_tester.framework.fixtures import image_based_manager
```

Comment:
While I'm not convinced our layout is as nice as it could be, while we have it in this structure we should probably make sure that something which isn't a bootstrap based test isn't in bootstrap_based_tests.

Reply:
Ignore this, I misread the github header.